src/pot.sol

```diff
@@ -138,7 +138,8 @@ contract Pot is LibNote {
 
     // --- Savings Rate Accumulation ---
     function drip() external note returns (uint tmp) {
-        require(now >= rho, "Pot/invalid-now");
+        if (now == rho) return chi;
+        require(now > rho, "Pot/invalid-now");
```

Comment:
Unrelated to this PR, shouldn't this be `invalid-rho`?

Reply:
time is relative...  Sorry, been a long day.